### DISCUSSION TITLE
Adding Trendline to Line Chart

### DISCUSSION THREAD

**Discussion by qlpanalytics** [apache/superset#18049] [Q&A / Help]
Hello,
We have been using Superset for about a year now and previously had been using Tableau. Everything with this platform has been great, however we've still been unable to find a way to add trendlines to line charts. I know there is a "Big Number with Trendline" chart type option, but that's not what we're looking for. We want to use the line chart view, and overlay a trendline on top of the data. Something similar to this:
![image](https://user-images.githubusercontent.com/88547703/149542061-ef333ecc-a453-47ec-8668-2c9d43792e5a.png)

Does anyone have any suggestions for how to get this functionality in Superset? Or if anyone has any coding experience around this, we would be interested in discussing a specialty project as well.

Thank you in advance for any help!

**Comment by nicosuave** [comment]
I wonder if it is possible to move this to Ideas? I'm afraid it might get overlooked otherwise

**Comment by lorransr** [comment]
Also, I believe it would be great to have a simple average line too

**Comment by ArmanKeswani** [comment]
Right click on the line chart and select "Show trend line" then select format after this select pane and then click on zero lines to choose the correct format.

**Reply by SrslyJosh** [reply]
@ArmanKeswani What version of Superset are you talking about? I do not see any special contextual menu when I right-click on charts, just the standard browser contextual menu.

**Reply by Aragonski97** [reply]
@SrslyJosh  I think he might be using Preset.
